libs/@blockprotocol/service/src/types.ts

```diff
@@ -241,21 +241,23 @@ export type MapboxRetrieveStaticMapData = {
  */
 export type MapboxRetrieveStaticMapResponseData = string;
 
+export type ServiceMessageError = "FORBIDDEN";
```

Comment:
Do we not want an `"UNIMPLEMENTED"` message? Or `"INTERNAL_ERROR"` like we have in the graph module?

Reply:
I believe the `service-module.json` should be updated to have these too?

Reply:
As a side note, if services do not add an error code, they don't even get the default ones – I briefly looked at changing that but it was going to mess around with `core` types so I decided not to.

Whoever picks up implementing these follow ups could look at that too, or discuss it with someone